packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -11,6 +11,7 @@ import defaults from 'lodash/defaults'
 import mapValues from 'lodash/mapValues'
 import zipWith from 'lodash/zipWith'
 import get from 'lodash/get'
+import debounce from 'lodash/debounce'
```

Comment:
I made this choice mainly because I have more trust in a widely used utility library than in our code 😄 (but also because we already includes lodash in all our projects, my choice would have been different otherwise)